src/qibo/backends/numpy.py

```diff
@@ -85,6 +85,11 @@ def zero_density_matrix(self, nqubits):
         state[0, 0] = 1
         return state
 
+    def identity_density_matrix(self, nqubits):
```

Comment:
Since you are adding this as a backend method, it would be good to also add and document it in the `AbstractBackend`.